### ISSUE TITLE
Stix export changed no according to standards. 

### ISSUE THREAD

**Issue by kanocool** [MISP/MISP#3754]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | No clue
| OS version (server)       | Ubuntu 16.04
| MISP version / git hash   | 2.4.96,

Im using a custom template which is made with the misp object standards, However when i export the event to stix it renders it wrong, i saw a similar issue. 
(https://github.com/MISP/MISP/issues/3482)

However Im not sure whether this seems to be and issue with the stix export as it did work with past versions. Any help is welcome.

### Expected behavior
<cybox:Observable id="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX">
						<cybox:Object id="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX">
							<cybox:Properties xsi:type="FileObj:FileObjectType">
								<FileObj:File_Name condition="Equals">XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</FileObj:File_Name>
								<FileObj:Hashes>
									<cyboxCommon:Hash>
										<cyboxCommon:Type condition="Equals" xsi:type="cyboxVocabs:HashNameVocab-1.0">MD5</cyboxCommon:Type>
										<cyboxCommon:Simple_Hash_Value condition="Equals">XXXXXXXXXXXXXXXXXXXXXXXXXXXXX</cyboxCommon:Simple_Hash_Value>
									</cyboxCommon:Hash>
								</FileObj:Hashes>
							</cybox:Properties>
						</cybox:Object>
					</cybox:Observable>

### Actual Behavior

<indicator:Observable id="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX">
                                        <cybox:Object id="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX">
                                            <cybox:Properties xsi:type="CustomObj:CustomObjectType">
                                                <cyboxCommon:Custom_Properties>
                                                    <cyboxCommon:Property name="Spear Phishing mail text: Additional-information">XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</cyboxCommon:Property>
                                                    <cyboxCommon:Property name="Spear Phishing mail domain|ip: Mail-server">XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</cyboxCommon:Property>
                                                    <cyboxCommon:Property name="Spear Phishing mail text: Behavioral-analysis">URL link to malicious Jar file</cyboxCommon:Property>
                                                    <cyboxCommon:Property name="Spear Phishing mail filename|sha256: Files-downloaded-attached-sha256">XXXXX|XXXXXXXXX</cyboxCommon:Property>
                                                    <cyboxCommon:Property name="Spear Phishing mail filename|sha256: Files-downloaded-attached-sha256">XXXXX|XXXXXXXXX</cyboxCommon:Property>
                                                    <cyboxCommon:Property name="Spear Phishing mail filename|sha256: Files-downloaded-attached-sha256">XXXXX|XXXXXXXXX </cyboxCommon:Property>

**Comment by chrisr3d** [comment]
Can you by any chance show us how your custom template looks like, so I can try to reproduce the behavior of the export script ?

**Comment by kanocool** [comment]
Hey, Sorry i forgot to close the issue, A collegue of mine has fixed it and pushed the solution. 
It has been fixed ever since, never saw the issue again after (I think) 2.4.98. It had something to do with the way the STIX/CYBOX closing tags were set in the output some readers are very picky with the syntax we solved it by looking at the output with the Stix and Cybox validators. Should you want to lookin to it.  I recommend to close the issue since its no longer an issue. :)
